approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# For development purposes, set the DEBUG_* variables to True in a 
+# local_settings.py file, which we .gitignore. This file should be added
+# in the same directory that contains this settings.py file.
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
```

Comment:
As we will have to set `DEBUG=False` in production anyway, it looks like we will have to set `ALLOWED_HOSTS` for security reasons.

I think settings.py should contain just `DEBUG=False` and the local_settings.py can contain `DEBUG=True` plus any extra settings needed. 

I was looking at the documentation for `TEMPLATE_DEBUG` and it looks like [it is deprecated](https://docs.djangoproject.com/en/1.8/ref/settings/#template-debug). I looked into the [`debug`](https://docs.djangoproject.com/en/1.8/topics/templates/#module-django.template.backends.django) setting in the `OPTIONS` hash of `TEMPLATES` and it defaults to the value of `DEBUG`, so I think we can safely remove `TEMPLATE_DEBUG`